javascript/javascript1/week2/index.js

```diff
@@ -0,0 +1,215 @@
+
+       /************************************************** 
+      *       1  :  Flight booking fullname function      *
+       ***************************************************/  
+
+           console.log('(1a) : Printing out full name by using returne \n\n ');
+
+     /*************************************************** 
+    * (1a) : Printing out full name by using return      *
+     ****************************************************/  
+
+
+
+
+var firstName='';
+let lastName='';
+function getFullName(firstName,lastName){
+  return firstName+' '+lastName;
+}
+let fullName=getFullName("Benjamin", "Hughes");
+console.log(fullName);
+
+
+console.log('\n\n (1b) :       Formal fullname  \n\n ');   
+
+
+    /********************************************* 
+    *  (1b) :       Formal fullname               *
+     *********************************************/   
+           
+
+let useFormalName = true;
```

Comment:
Do you need this?

Reply:
i did it because we need it in getFullName() as parameter... so we have to declare this variable first,,,